docs/source/resources/frequently_asked_questions.ipynb

```diff
@@ -785,6 +785,10 @@
     "\n",
     "You can run DFS on a single table. Featuretools will be able to generate features for your data, but only transform features.\n",
```

Comment:
We need to expand on this and better highlight why single table isn't recommended